app/src/util_ios.mm

```diff
@@ -161,21 +161,20 @@ void ForEachAppDelegateClass(void (^block)(Class)) {
   return array;
 }
 
-void NSArrayOfNSStringToVectorOfString(NSArray* array, std::vector<std::string>* string_vector) {
+void NSArrayOfNSStringToVectorOfString(NSArray *array, std::vector<std::string> *string_vector) {
```

Comment:
Apparently I branched too soon..?  The only changes to these files is their format.

Reply:
clang-format will generally respect the first "pointer gravity" (left or right) it sees in the file, and enforce it throughout.

Reply:
Got it.